contracts/pool/lib.rs

```diff
@@ -101,6 +118,7 @@ pub mod contract {
         value: Balance,
     }
 
+    /// Event: Delegation Power is changed
```

Comment:
What is Delegation Power? Delegation Power can be difficult to understand.
Looking at the use case, this field is for borrow_for, this control the quantity of tokens that can be borrowed by other specified user?
```rust
// logics/impls/pool/mod.rs
    #[modifiers(delegated_allowed(borrower, borrow_amount))]
    default fn borrow_for(&mut self, borrower: AccountId, borrow_amount: Balance) -> Result<()> {
```